dbms/src/Functions/GeoUtils.cpp

```diff
@@ -60,11 +66,10 @@ inline UInt8 singleCoordBitsPrecision(UInt8 precision, CoordType type)
 inline Encoded encodeCoordinate(Float64 coord, Float64 min, Float64 max, UInt8 bits)
 {
     Encoded result;
-    result.fill(0);
```

Comment:
std::array doesn't initialize values by default, so the higher bits are going to be uninitialized. Probably they are not used, but I'd prefer to not have uninitialized values. Is this initialization becoming a performance bottleneck? This encoding looks suboptimal, it basically does the shift and division bit-by-bit. I think we can optimize it if needed. But this should be a different PR.

I'll remove this change myself if you don't mind, so you don't have to update the PR.